README.md

```diff
@@ -1,4 +1,6 @@
-# Kapitan: Generic templated configuration management for Kubernetes, Terraform and other things
+# Kapitan <img src="./docs/images/kapitan_logo.png" width="250" align="right">
```

Comment:
Img doesn't look great with right align, please leave it like before :)

Reply:
Done!